src/style/variables.scss

```diff
@@ -13,8 +13,6 @@ $iui-component-height-small: $iui-baseline * 3 - $iui-component-offset * 2; // 2
 $iui-component-height-large: $iui-baseline * 5 - $iui-component-offset * 2; // 49px
 
 $iui-component-padding-horizontal: $iui-sm; // 12px
-$iui-component-padding-horizontal-small: $iui-sm * 0.5; // 6px
```

Comment:
Are these really not needed anymore?

Reply:
They are needed if we want different left / right padding for small, medium, large sizes.  If we choose to have all 3 sizes match like the PR does currently, the large & small variables aren't needed anymore.

Reply:
I asked because you have used multiplication in buttons:
![image](https://user-images.githubusercontent.com/36186912/124434375-78f5f400-dd7c-11eb-8689-ff06e5087167.png)

So maybe we need to have `small` and `large` variables here just to avoid different sizing in the future.

Reply:
I've added those variables back in exclusively for the button to use now.